xbmc/settings/dialogs/GUIDialogSettingsManualBase.cpp

```diff
@@ -966,6 +984,22 @@ ISettingControl* CGUIDialogSettingsManualBase::GetButtonControl(const std::strin
   return control;
 }
 
+ISettingControl* CGUIDialogSettingsManualBase::GetInfoButtonControl(const std::string &format, bool delayed /* = false */, int heading /* = -1 */, bool hideValue /* = false */)
+{
+  CSettingControlInfoButton *control = new CSettingControlInfoButton();
+  if (!control->SetFormat(format))
+  {
+    delete control;
+    return NULL;
+  }
+
+  control->SetDelayed(delayed);
```

Comment:
I can remove this as wanted on other request, but has seen that it is also used on all other types. Please give me a comment.